src/main/js/helpers/axis.js

```diff
@@ -191,37 +191,136 @@ const getAxesScale = (axis, scale, config) => {
         getAxisInfoOrientation(config.axis.x.orientation)
     );
 
-    // If ticksCount is undefined or greater than TICKS_MAXCOUNT
-    // AND if the Y2 is visible, then utilize a default value for
-    // the ticksCount. This is based on the ranges of the Y & Y2 axes.
+    if (hasY2Axis(config.axis)) {
+        if (!utils.isUndefined(config.axis.y.ticks.values)) {
```

Comment:
You can add a comment here if the y-axis tick values are provided, it will take the priority

Reply:
updated comments:
https://github.com/cerner/carbon-graphs/pull/249/commits/6533ea2824a5399bd0de7bfab38eb879c14af30f